src/razor/src/document/razorDocumentManager.ts

```diff
@@ -189,7 +190,7 @@ export class RazorDocumentManager implements IRazorDocumentManager {
             }
         }
 
-        this.notifyDocumentChange(document, RazorDocumentChangeKind.closed);
+        this.notifyDocumentChange(document, RazorDocumentChangeKind.closed, []);
```

Comment:
intentionally leaving these empty. For open/close/add/remove the changes are irrelevant.